charts/openshift-etcd-backup/Chart.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+name: openshift-etcd-backup
+description: Chart for openshfit-etcd-backup solution
+type: application
+version: 0.1.0
+appVersion: 1.0.0
```

Comment:
There's not a single tagged release available on [adfinis-sygroup/openshift-etcd-backup](https://github.com/adfinis-sygroup/openshift-etcd-backup), so I'm not sure if we should use 1.0.0 as the appVersion.

Reply:
what do you suggest? should we use the same as `version` then?

Reply:
We should release a version upstream and as @hairmare recommended build our own image based on the OpenShift ART image.

Reply:
@tongpu see [https://github.com/adfinis-sygroup/helm-charts/pull/287/files#r656032602 #r656032602] for the reason behind the usage of the upstream image.

Reply:
container images are for application delivery, configmaps are for configs

if need be we can push our image to quay so it will come from the same source as the image used upstream

Reply:
> if need be we can push our image to quay so it will come from the same source as the image used upstream

No, the image originates from the internal mirror registry not from quay in this case.

Reply:
> container images are for application delivery, configmaps are for configs
> 
> if need be we can push our image to quay so it will come from the same source as the image used upstream

@hairmare This is image should™ always be present in the image cache on every OpenShift node, because it also used for some internal OpenShift shenanigans.